protocols/mdns/src/behaviour.rs

```diff
@@ -49,6 +49,8 @@ lazy_static! {
         SocketAddr::from((Ipv4Addr::new(224, 0, 0, 251), 5353));
 }
 
+/// Configuration for mDNS.
+#[derive(Clone, Debug)]
```

Comment:
Could you extract this change to a separate pull request? Making only a single cohesive change per pull request makes maintaining this project a lot easier, especially around version and changelog management.

Reply:
I see you renamed the PR, I moved the api changes, as this one has the branch name `mdns-config`